public/app/features/trails/ActionTabs/MetricOverviewScene.tsx

```diff
@@ -67,26 +64,27 @@ export class MetricOverviewScene extends SceneObjectBase<MetricOverviewSceneStat
   public static Component = ({ model }: SceneComponentProps<MetricOverviewScene>) => {
     const { metadata, metadataLoading } = model.useState();
     const variable = model.getVariable();
-    const { loading: labelsLoading } = variable.useState();
-    const labelOptions = getLabelOptions(model, variable).filter((l) => l.value !== ALL_VARIABLE_VALUE);
+    const { loading: labelsLoading, options: labelOptions } = variable.useState();
+
+    const help = metadata?.help;
+    const type = metadata?.type;
+    const unit = metadata?.unit;
```

Comment:
Since the metadata is an optional value, we might end up having those as `undefined`

Reply:
Not changing the metadata part in this PR, but good point